lib/src/utils/index.ts

```diff
@@ -22,7 +22,7 @@ function getSectionName(id: string | null, isSectionId?: boolean): Maybe<string>
     // If the id is shaped like root_section_id
     return idParts[1];
   }
-  if (isSectionId && id !== 'root') {
+  if ((isSectionId && id !== 'root') || (id && id.length > 0)) {
```

Comment:
Can you store the condition statement in a variable to help reader to understand what are you testing ?

Furthermore, are you sure about the test `id && id.length > 0` because if `id=root` the condition should return `true` and I'm not sure we want this behavior. At least we should write a test to ensure I'm wrong.

Reply:
Ouh my bad, the logic is fine, it was the comment that wasn't and leads you to confusion.

Reply:
Write a test please to confirm this please.